assets/js/atomic/blocks/product-elements/sku/supports.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { isFeaturePluginBuild } from '@woocommerce/block-settings';
+import { __experimentalGetSpacingClassesAndStyles } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import sharedConfig from '../shared/config';
+
+export const supports = {
+	...sharedConfig.supports,
+	...( isFeaturePluginBuild() && {
+		color: {
+			text: true,
+			background: true,
+		},
+		typography: {
+			fontSize: true,
+			lineHeight: true,
```

Comment:
If we can remove `__experimentalSelector`, then we could make these styles available in WC core as well, so only the experimental ones are gated to the feature plugin. :slightly_smiling_face:

Reply:
That makes sense. I will make the necessary changes 🤝